Newlookwithoutssl

krishnatejaperannagari opened the pull request.

krishnatejaperannagari committed:
soc runnig buildfarm

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
tables complete

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
new look that suits all buildfarm pages

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
form positioning complete

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
new look reviews mistakes rectified

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
new look adjustment when data is available

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
some minor changes

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
Merge git://git.samba.org/build-farm into newlook

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
deleting changes directory

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
adding submodule

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
using ssi for the html pages in submoule samba-web

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
few changes for using samba web submodule

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
breaking files,deleting unnecessary and using submodule without virtual

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
using urllib and parsing

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
changing html to cgi pages

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
deleting submodule and adding the main samba one and using submodule

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
new module

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
modifications

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
Merge https://github.com/jelmer/build-farm into newlookwithoutssl

krishnatejaperannagari committed:
changing buttons to match new layout

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
reviews

krishnatejaperannagari committed:
removing empty <pre></pre> pairs

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
Merge https://github.com/jelmer/build-farm into newlookwithoutssl

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
reviews

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
changes to sambaweb and content

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
reviews

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
modifications to use footer and search field

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
serach using functions as dict

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

jelmer commented on buildfarm/web/__init__.py:
```diff
@@ -971,22 +1021,33 @@ def html_page(self, form, lines):
-        yield util.FileLoad(os.path.join(webdir, "footer.html"))
-        yield "</body>"
-        yield "</html>"
+        yield util.SambaWebFileLoad(os.path.join(webdir, "samba-web"), "menu_sitemap_samba_closed.html")
```
There doesn't seem to be a file with this name in samba-web ?

Did you mean to add a file in web/ instead?

krishnatejaperannagari replied:
ya i sent you a mail earlier about changing index.html of samba -web to use sitemap from this file this is for that. sending mail again

jelmer replied:
On Sat, Aug 09, 2014 at 03:15:19AM -0700, krishnatejaperannagari wrote:

> > ```
> >      yield "".join(lines)
> > ```
> > -        yield util.FileLoad(os.path.join(webdir, "footer.html"))
> > -        yield "</body>"
> > -        yield "</html>"
> > -        yield util.SambaWebFileLoad(os.path.join(webdir, "samba-web"), "menu_sitemap_samba_closed.html")
> 
> ya i sent you a mail earlier about changing index.html f samba -web to use sitemap from this file this is for that. sending mail again
> Do we need this at all? None of the other pages on samba.org have this
> menu as far as I can tell.

Cheers,

Jelmer

## 

Jelmer Vernooij jelmer@samba.org - https://jelmer.uk/

krishnatejaperannagari replied:
it looks good thats why even without that its okay u can just uncomment this line if u dont need it and add it whenever u make changes to the samba web

On Sat, Aug 9, 2014 at 4:40 PM, Jelmer Vernooij notifications@github.com
wrote:

> In buildfarm/web/**init**.py:
> 
> > ```
> >      yield "".join(lines)
> > ```
> > -        yield util.FileLoad(os.path.join(webdir, "footer.html"))
> > -        yield "</body>"
> > -        yield "</html>"
> > -        yield util.SambaWebFileLoad(os.path.join(webdir, "samba-web"), "menu_sitemap_samba_closed.html")
> 
> On Sat, Aug 09, 2014 at 03:15:19AM -0700, krishnatejaperannagari wrote: >
> yield "".join(lines) > - yield util.FileLoad(os.path.join(webdir,
> "footer.html")) > - yield "</body>" > - yield "</html>" > + yield
> util.SambaWebFileLoad(os.path.join(webdir, "samba-web"),
> "menu_sitemap_samba_closed.html") ya i sent you a mail earlier about
> changing index.html f samba -web to use sitemap from this file this is for
> that. sending mail again
> Do we need this at all? None of the other pages on samba.org have this
> menu as far as I can tell. Cheers, Jelmer
> … <#147ba76216c2f871_>
> -- Jelmer Vernooij jelmer@samba.org - https://jelmer.uk/
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/jelmer/build-farm/pull/12/files#r16025537.

## 

krishna teja perannagari

krishnatejaperannagari committed:
uncommenting sitemap part

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
removing empty pre obtained by using re search

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
using sitemap if necessarychanges and changes to form select

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

jelmer commented on buildfarm/web/__init__.py:
```diff
@@ -971,22 +1018,34 @@ def html_page(self, form, lines):
         yield "    <link rel='stylesheet' href='/build_farm.css' type='text/css' media='all'/>"
-        yield "    <link rel='stylesheet' href='//www.samba.org/samba/style/common.css' type='text/css' media='all'/>"
-        yield "    <link rel='shortcut icon' href='//www.samba.org/samba/images/favicon.ico'/>"
+        yield "    <link rel='stylesheet' href='/newstyles.css' type='text/css' media='all'/>"
```
What is the difference between newstyles.css and build_farm.css ?

krishnatejaperannagari replied:
its all the new css i have written didnt do much changes in  build_farm.css all done in newstyle.css

jelmer replied:
On Wed, Aug 13, 2014 at 02:37:02AM -0700, krishnatejaperannagari wrote:

> > @@ -971,22 +1018,34 @@ def html_page(self, form, lines):
> >          yield "    <meta name='description' contents='Home of the Samba Build Farm, the automated testing facility.'/>\n"
> >          yield "    <meta name='robots' contents='noindex'/>"
> >          yield "    <link rel='stylesheet' href='/build_farm.css' type='text/css' media='all'/>"
> > -        yield "    <link rel='stylesheet' href='//www.samba.org/samba/style/common.css' type='text/css' media='all'/>"
> > -        yield "    <link rel='shortcut icon' href='//www.samba.org/samba/images/favicon.ico'/>"
> > -        yield "    <link rel='stylesheet' href='/newstyles.css' type='text/css' media='all'/>"
> 
> its all the new css i have written didnt do much changes in  build_farm.css all done in newstyle.css

Can you please integrate them into build_farm.css, so we just have one
CSS file for the build farm?

Jelmer

jelmer commented on web/build_farm.css:
```diff
@@ -1,227 +1,233 @@
+	padding-top:4px;
+	background-color:#FFFF64;
+}
+.stripe {
```
Please make sure you use unix-style line-endings. using dos line endings breaks github, and makes this file hard to review.

krishnatejaperannagari replied:
i am using ubuntu linux and gedit for editing css is there anything i need to change

jelmer replied:
On Wed, Aug 13, 2014 at 02:38:53AM -0700, krishnatejaperannagari wrote:

> > -        margin-top:-42px;
> > -        margin-left:1px;
> > -        z-index:1;  
> >   +}
> >   +
> >   +#banner {
> > -   position:absolute;
> > -   top:0;
> > -   left:0;
> > -        text-align: center;
> > -   height:90px;
> > -   width:100%;
> > -   padding-top:4px;
> > -   background-color:#FFFF64;
> >   +}
> >   +.stripe {
> 
> i am using ubuntu linux and gedit for editing css is there anything i need to change

You can see that there are DOS newlines in the file by looking at the github
diff (^M).

To convert the file from DOS line endings to Unix line endings, you can use
the dos2unix tool from the dos2unix package.

Cheers,

Jelmer

jelmer commented on web/newstyles.css:
```diff
@@ -0,0 +1,205 @@
+}
+
+.relatedLeftBottom{
+background-image: url(bgBoxSpecialBottomNew.gif);
```
Can we use the stylesheet from samba-web here, and the gifs from that repository?

krishnatejaperannagari replied:
ya done using background-size

jelmer commented:
Where do we use icon_hide_24.png ? In the code I only see references to icon_hide_16.png.

jelmer commented:
I've added some more comments. Sorry they're all coming in doses. This is a large branch and it makes a lot of nonobvious, which makes it hard to review :(

krishnatejaperannagari committed:
using bgboxspecial from sambaweb

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari commented:
actually there were two icons one with 16px and the other one with 24 px in
the original buildfarm before starting the work so i have replaced both
even i didnt see icon_hide_24.png used anywhere
i have added it for i dont know whats will happen if i remove it

On Wed, Aug 13, 2014 at 1:19 AM, Jelmer Vernooij notifications@github.com
wrote:

> Where do we use icon_hide_24.png ? In the code I only see references to
> icon_hide_16.png.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/jelmer/build-farm/pull/12#issuecomment-51967288.

## 

krishna teja perannagari

krishnatejaperannagari commented:
thats okay i will make changes as fast as i can

On Wed, Aug 13, 2014 at 1:20 AM, Jelmer Vernooij notifications@github.com
wrote:

> I've added some more comments. Sorry they're all coming in doses. This is
> a large branch and it makes a lot of nonobvious, which makes it hard to
> review :(
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/jelmer/build-farm/pull/12#issuecomment-51967383.

## 

krishna teja perannagari

krishnatejaperannagari committed:
removing unnecessaryspaces

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
using dos2nix

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
merging stylesheets

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

krishnatejaperannagari committed:
Merge https://github.com/jelmer/build-farm into newlookwithoutssl

merging to resolveconflicts

krishnatejaperannagari committed:
final changes

Signed-off-by: krishnatejaperannagari <krishnatejaperannagari@gmail.com>

jelmer commented on web/build_farm.css:
```diff
@@ -225,3 +231,214 @@ div.build-section {
+	float: right;
+	height: 139px;
+	width: 100%;
+	background-image: url(newheader.png);
```
Can this image be loaded from samba-web instead?

krishnatejaperannagari replied:
done

jelmer commented:
Where do the icon images come from, what is their license?

jelmer commented:
I think those are my final points. Thanks for your patience!
